backend/push.js

```diff
@@ -199,11 +199,19 @@ export const postEvent = async (subscription: Object, event: ?string): Promise<v
 
   if (!req.ok) {
     const endpointHost = new URL(subscription.endpoint).host
-    console.warn(`Error ${req.status} sending push notification to '${subscription.id}' via ${endpointHost}`)
+    console.warn(
+      await req.text().then(text => {
+        try {
+          return (text && JSON.parse(text)) || '(empty response)'
```

Comment:
Why not just log the text as is? I don't think we should be parsing this as JSON.

Reply:
Because it sends back JSON and the logs will look nice this way:

![Screenshot 2025-02-26 at 10 32 02 AM](https://github.com/user-attachments/assets/6e2b03de-f48f-4c2d-89d3-6c0b150e61ef)

Reply:
It only slows things down imperceptibly on exception... I think it'll be OK.

The prettiness only happens if the sysadmin wants it (the actual logs are raw JSON).

Reply:
Also, if you're making it a PR, probably the payload should be logged at a different level (e.g., debug or info) to avoid filling up logs.

Reply:
I'm aware the prettiness only happens if the sysadmin wants it. But, you're not guaranteed that the response will be JSON to begin with, and it can be controlled by an adversary (and because of this, it can happen pretty often, as often as the adversary chooses, actually). There's no need to parse the payload, and the sysadmin can use `jq` if they want  pretty output.